bai7.js

```diff
@@ -0,0 +1,11 @@
+const mapArray = (arr, callbackFunction) => {
+  return callbackFunction(arr);
```

Comment:
It's seem that you are missunderstanding the ex `7. Write a function mapArray that takes an array and a callback function. It should return a new array with the result of applying the callback function to each element.`

The idea:  `return arr.map((item) => callbackFunction(item));`